setup.py

```diff
@@ -34,14 +34,14 @@
         "onnxruntime>=1.9.0",
         "datasets>=1.2.1",
         "evaluate",
-        "protobuf==3.20.1",
+        "protobuf>=3.20.1",
```

Comment:
Shouldn't we put `>=3.20.2` since v3.20.1 is the version with the security issue?

Reply:
Not sure, maybe other package require 3.20.1.

Reply:
Fair enough